examples/twr_aloha/twr_shell.c

```diff
@@ -78,6 +78,8 @@ int _twr_ifconfig(int argc, char **argv)
     return 0;
 }
 
+SHELL_COMMAND(ifconfig, "Network interface information", _twr_ifconfig);
```

Comment:
`_twr_ifconfig` could now be `static`. (And it actually must be, as a single leading underscore is reserved in global scope